<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>CSU-Bioinfomatics Group</title>
<meta name="robots" content="all" />
<meta name="keywords" content="CSU，Bioinformatic" />
<meta name="description" content="CSU，Bioinformatic。" />
<link href="../css/reset.css" rel="stylesheet" type="text/css" />
<link href="../css/webmain.css" rel="stylesheet" type="text/css" />
<link href="../css/ddsmoothmenu.css" rel="stylesheet" type="text/css" />
<link href="../css/expension.css" rel="stylesheet" type="text/css" />
<link href="../css/friendLink.css" rel="stylesheet" type="text/css" />
<link href="../css/achievements.css" rel="stylesheet" type="text/css" />
<script type="text/javascript" src="../scripts/jquery-1.4.2.min.js"></script>
<script type="text/javascript" src="../scripts/jquery.KinSlideshow-1.2.1.js"></script>
<script type="text/javascript" src="../scripts/webtry_roll.js"></script>
<script type="text/javascript" src="../scripts/ddsmoothmenu.js"></script>
<script type="text/javascript" src="../scripts/achievements.js"></script>
<script type="text/javascript">
ddsmoothmenu.init({
	mainmenuid: "MainMenu", //menu DIV id
	orientation: 'h', //Horizontal or vertical menu: Set to "h" or "v"
	classname: 'ddsmoothmenu', //class added to menu's outer DIV
	//customtheme: ["#1c5a80", "#18374a"],
	contentsource: "markup" //"markup" or ["container_id", "path_to_menu_file"]
})
</script>
</head>
<body>
<div id="wrapper">
    <div class="top">
    <img src="../images/logo.gif" width="650" height="90" alt="CSU Bioinformatics Group" />
    <div id="lang"><img src="../images/gb.png" width="45" height="45" alt="visits" /><span class="STYLE1">Visits: 100</span></div>
  </div>
  <div id="MainMenu" class="ddsmoothmenu">
		<ul>
		  <li><a href="../index.html" title="Home Page"><span>Home</span></a></li>
		  <li><a href="../members.html" title="Group Members"><span>Group Members</span></a></li>
		  <li><a href="../project.html" title="Projects"><span>Projects</span></a></li>
		  <li><a href="../achievements.html" title="Achievements" id="menu_selected"><span>Achievements</span></a></li>
		  <li><a href="../download.html" title="Download"><span>Download</span></a></li>
		  <li><a href="../contact.html" title="Contact Us"><span>Contact</span></a></li>
		  <!--<li><a href="news.html" title="Contact Us"><span>Contact</span></a></li>
		  <li><a href="guestbook.html" title="留言反馈"><span>1</span></a></li>
		  <li><a href="http://www.netgather.com" target="_blank" title="网聚网"><span>网聚网</span></a></li>-->
		</ul>
  </div>
<script type="text/javascript">
$(function(){
    $("#banner").KinSlideshow({
            moveStyle:"right",
            titleBar:{titleBar_height:32,titleBar_bgColor:"#000",titleBar_alpha:0.7},
            titleFont:{TitleFont_size:12,TitleFont_color:"#FFFFFF",TitleFont_weight:"normal"},
            btn:{btn_bgColor:"#2d2d2d",btn_bgHoverColor:"#1072aa",btn_fontColor:"#FFF",btn_fontHoverColor:"#FFF",btn_borderColor:"#4a4a4a",btn_borderHoverColor:"#1188c0",btn_borderWidth:1}
    });
})
</script>
  <div id="banner">
    <a href="#"><img src="../images/banner01.jpg" alt="为生物信息学中关键与热点问题提供新的理论基础与研究方法，以推动我国生物信息学研究以及计算机基础理论研究的发展！" width="950" height="152" /></a>
    <a href="#"><img src="../images/banner02.jpg" alt="为生物信息学中关键与热点问题提供新的理论基础与研究方法，以推动我国生物信息学研究以及计算机基础理论研究的发展！" width="950" height="152" /></a>
    <a href="#"><img src="../images/banner03.jpg" alt="为生物信息学中关键与热点问题提供新的理论基础与研究方法，以推动我国生物信息学研究以及计算机基础理论研究的发展！" width="950" height="152" /></a>
  </div>
  <div id="page_main" class="clearfix">
    <div class="page-right">
      <div class="site-nav"><span>Current Location : </span><a href="../achievements.html">Achievements</a> &gt;&gt; Conference Papers</div>  
      	<!-- S -->
		<div id="content"> 
			<span id="key1" class="key" onClick="shoppingcat('1');">Abstract</span>
			1. Greedily Mining l-dense Subgraphs in Protein Interaction Networks
		</div>
		<div id="intro1" class="intro">
			<hr>作者：Li Min, Wang Jianxin, Chen Jian'er<br />期刊：The 9th International Conference for Young Computer Scientists, 2008. ICYCS 2008., pp 1018-1023 
			<br />日期：2008-11-18
			<br />&nbsp;&nbsp;<font size="+1">Abstract:</font> Dense subgraphs of protein interaction networks are believed to be potential protein complexes and play an important role in analyzing cellular organization and predicting functions of proteins. In this paper, we present a new algorithm LD-Miner for mining l-dense subgraphs in protein interaction networks. We apply algorithm LD-Miner to the protein interaction network of Saccharomyces cerevisiae collected from MIPS. Many well known protein complexes annotated by manually are detected. We also match our predicted clusters to the known complexes systematically analyzed by Given et al., Ho et al., and Krogan et al., compared to other previous density-based algorithms, LD-Miner matches much more experimentally determined complexes and known complexes by systematic analysis. Moreover, LD-Miner is very fast, which can be used in even larger biological networks.
		</div>
		
		<div id="content"> 
			<span id="key2" class="key" onClick="shoppingcat('2');">Abstract</span>
			2. A model of higher accuracy for the individual haplotyping problem based on weighted SNP fragments and genotype with errors 
		</div>
		<div id="intro2" class="intro">
			<hr>作者：Xie, M.Wang, J.Chen, J.<br />期刊：Bioinformatics 24, pp. i105-i113, 2008  
			<br />日期：2008-10-01
			<br />&nbsp;&nbsp;<font size="+1">Abstract:</font>
MOTIVATION: In genetic studies of complex diseases, haplotypes provide more information than genotypes. However, haplotyping is much more difficult than genotyping using biological techniques. Therefore effective computational techniques have been in demand. The individual haplotyping problem is the computational problem of inducing a pair of haplotypes from an individual's aligned SNP fragments. Based on various optimal criteria and including different extra information, many models for the problem have been proposed. Higher accuracy of the models has been an important issue in the study of haplotype reconstruction.
<br />
RESULTS: The current article proposes a highly accurate model for the single individual haplotyping problem based on weighted fragments and genotypes with errors. The model is proved to be NP-hard even with gapless fragments. Based on the characteristics of Single Nucleotide Polymorphism (SNP) fragments, a parameterized algorithm of time complexity O(nk(2)2(k(2)) + m log m + mk(1)) is developed, where m is the number of fragments, n is the number of SNP sites, k(1) is the maximum number of SNP sites that a fragment covers (no more than n and usually smaller than 10) and k(2) is the maximum number of the fragments covering a SNP site (usually no more than 19). Extensive experiments show that this model is more accurate in haplotype reconstruction than other models.
<br />
AVAILABILITY: The program of the parameterized algorithm can be obtained by sending an email to the corresponding author.
		</div>
		
		<div id="content"> 
			<span id="key1" class="key" onClick="shoppingcat('1');">Abstract</span>
			3. Greedily Mining l-dense Subgraphs in Protein Interaction Networks
		</div>
		<div id="intro1" class="intro">
			<hr>作者：Li Min, Wang Jianxin, Chen Jian'er<br />期刊：The 9th International Conference for Young Computer Scientists, 2008. ICYCS 2008., pp 1018-1023 
			<br />日期：2008-11-18
			<br />&nbsp;&nbsp;<font size="+1">Abstract:</font> Dense subgraphs of protein interaction networks are believed to be potential protein complexes and play an important role in analyzing cellular organization and predicting functions of proteins. In this paper, we present a new algorithm LD-Miner for mining l-dense subgraphs in protein interaction networks. We apply algorithm LD-Miner to the protein interaction network of Saccharomyces cerevisiae collected from MIPS. Many well known protein complexes annotated by manually are detected. We also match our predicted clusters to the known complexes systematically analyzed by Given et al., Ho et al., and Krogan et al., compared to other previous density-based algorithms, LD-Miner matches much more experimentally determined complexes and known complexes by systematic analysis. Moreover, LD-Miner is very fast, which can be used in even larger biological networks.
		</div>
		
		<div id="content"> 
			<span id="key1" class="key" onClick="shoppingcat('1');">Abstract</span>
			4. Greedily Mining l-dense Subgraphs in Protein Interaction Networks
		</div>
		<div id="intro1" class="intro">
			<hr>作者：Li Min, Wang Jianxin, Chen Jian'er<br />期刊：The 9th International Conference for Young Computer Scientists, 2008. ICYCS 2008., pp 1018-1023 
			<br />日期：2008-11-18
			<br />&nbsp;&nbsp;<font size="+1">Abstract:</font> Dense subgraphs of protein interaction networks are believed to be potential protein complexes and play an important role in analyzing cellular organization and predicting functions of proteins. In this paper, we present a new algorithm LD-Miner for mining l-dense subgraphs in protein interaction networks. We apply algorithm LD-Miner to the protein interaction network of Saccharomyces cerevisiae collected from MIPS. Many well known protein complexes annotated by manually are detected. We also match our predicted clusters to the known complexes systematically analyzed by Given et al., Ho et al., and Krogan et al., compared to other previous density-based algorithms, LD-Miner matches much more experimentally determined complexes and known complexes by systematic analysis. Moreover, LD-Miner is very fast, which can be used in even larger biological networks.
		</div>
		
		<div id="content"> 
			<span id="key1" class="key" onClick="shoppingcat('1');">Abstract</span>
			5. Greedily Mining l-dense Subgraphs in Protein Interaction Networks
		</div>
		<div id="intro1" class="intro">
			<hr>作者：Li Min, Wang Jianxin, Chen Jian'er<br />期刊：The 9th International Conference for Young Computer Scientists, 2008. ICYCS 2008., pp 1018-1023 
			<br />日期：2008-11-18
			<br />&nbsp;&nbsp;<font size="+1">Abstract:</font> Dense subgraphs of protein interaction networks are believed to be potential protein complexes and play an important role in analyzing cellular organization and predicting functions of proteins. In this paper, we present a new algorithm LD-Miner for mining l-dense subgraphs in protein interaction networks. We apply algorithm LD-Miner to the protein interaction network of Saccharomyces cerevisiae collected from MIPS. Many well known protein complexes annotated by manually are detected. We also match our predicted clusters to the known complexes systematically analyzed by Given et al., Ho et al., and Krogan et al., compared to other previous density-based algorithms, LD-Miner matches much more experimentally determined complexes and known complexes by systematic analysis. Moreover, LD-Miner is very fast, which can be used in even larger biological networks.
		</div>
		
		<div id="content"> 
			<span id="key1" class="key" onClick="shoppingcat('1');">Abstract</span>
			6. Greedily Mining l-dense Subgraphs in Protein Interaction Networks
		</div>
		<div id="intro1" class="intro">
			<hr>作者：Li Min, Wang Jianxin, Chen Jian'er<br />期刊：The 9th International Conference for Young Computer Scientists, 2008. ICYCS 2008., pp 1018-1023 
			<br />日期：2008-11-18
			<br />&nbsp;&nbsp;<font size="+1">Abstract:</font> Dense subgraphs of protein interaction networks are believed to be potential protein complexes and play an important role in analyzing cellular organization and predicting functions of proteins. In this paper, we present a new algorithm LD-Miner for mining l-dense subgraphs in protein interaction networks. We apply algorithm LD-Miner to the protein interaction network of Saccharomyces cerevisiae collected from MIPS. Many well known protein complexes annotated by manually are detected. We also match our predicted clusters to the known complexes systematically analyzed by Given et al., Ho et al., and Krogan et al., compared to other previous density-based algorithms, LD-Miner matches much more experimentally determined complexes and known complexes by systematic analysis. Moreover, LD-Miner is very fast, which can be used in even larger biological networks.
		</div>
		
		<div id="content"> 
			<span id="key1" class="key" onClick="shoppingcat('1');">Abstract</span>
			7. Greedily Mining l-dense Subgraphs in Protein Interaction Networks
		</div>
		<div id="intro1" class="intro">
			<hr>作者：Li Min, Wang Jianxin, Chen Jian'er<br />期刊：The 9th International Conference for Young Computer Scientists, 2008. ICYCS 2008., pp 1018-1023 
			<br />日期：2008-11-18
			<br />&nbsp;&nbsp;<font size="+1">Abstract:</font> Dense subgraphs of protein interaction networks are believed to be potential protein complexes and play an important role in analyzing cellular organization and predicting functions of proteins. In this paper, we present a new algorithm LD-Miner for mining l-dense subgraphs in protein interaction networks. We apply algorithm LD-Miner to the protein interaction network of Saccharomyces cerevisiae collected from MIPS. Many well known protein complexes annotated by manually are detected. We also match our predicted clusters to the known complexes systematically analyzed by Given et al., Ho et al., and Krogan et al., compared to other previous density-based algorithms, LD-Miner matches much more experimentally determined complexes and known complexes by systematic analysis. Moreover, LD-Miner is very fast, which can be used in even larger biological networks.
		</div>
		
		<div id="content"> 
			<span id="key1" class="key" onClick="shoppingcat('1');">Abstract</span>
			8. Greedily Mining l-dense Subgraphs in Protein Interaction Networks
		</div>
		<div id="intro1" class="intro">
			<hr>作者：Li Min, Wang Jianxin, Chen Jian'er<br />期刊：The 9th International Conference for Young Computer Scientists, 2008. ICYCS 2008., pp 1018-1023 
			<br />日期：2008-11-18
			<br />&nbsp;&nbsp;<font size="+1">Abstract:</font> Dense subgraphs of protein interaction networks are believed to be potential protein complexes and play an important role in analyzing cellular organization and predicting functions of proteins. In this paper, we present a new algorithm LD-Miner for mining l-dense subgraphs in protein interaction networks. We apply algorithm LD-Miner to the protein interaction network of Saccharomyces cerevisiae collected from MIPS. Many well known protein complexes annotated by manually are detected. We also match our predicted clusters to the known complexes systematically analyzed by Given et al., Ho et al., and Krogan et al., compared to other previous density-based algorithms, LD-Miner matches much more experimentally determined complexes and known complexes by systematic analysis. Moreover, LD-Miner is very fast, which can be used in even larger biological networks.
		</div>
		<!-- E -->
      
    </div>
    <div class="page-left">
      <div class="index-news">
        <h2><span>Scientific Research Achievements</span></h2>
        <div id="LeftMenu" class="ddsmoothmenu-v">
          <ul>
			<li><a href="JournalPaper.html" title="JournalPaper"><span>Journal Papers</span></a></li>
			<li><a href="ConferencePaper.html" title="ConferencePaper"><span>Conference Papers</span></a></li>
			<li><a href="Patent.html" title="Patent"><span>Patents</span></a></li>
			<li><a href="SoftwareCopyright.html" title="SoftwareCopyright"><span>Software Copyright</span></a></li>
          </ul>
        </div>
        <script type="text/javascript">
		ddsmoothmenu.init({
			mainmenuid: "LeftMenu", //Menu DIV id
			orientation: 'v', //Horizontal or vertical menu: Set to "h" or "v"
			classname: 'ddsmoothmenu-v', //class added to menu's outer DIV
			//customtheme: ["#804000", "#482400"],
			contentsource: "markup" //"markup" or ["container_id", "path_to_menu_file"]
			})
        </script>
      </div>
      <div class="left-contact">
        <h2><span>联系我们</span></h2>
        <div class="xsnazzy">
			<b class="xtop"><b class="xb1"></b><b class="xb2 color_a">
			</b><b class="xb3 color_a"></b><b class="xb4 color_a"></b></b>
			<div class="xboxcontent">
				<h1 class="color_a"><a href="http://www.csu.edu.cn/" target="_blank">中南大学</a></h1>
				<p>Central South University</p>
			</div>
			<b class="xbottom"><b class="xb4"></b><b class="xb3"></b>
			<b class="xb2"></b><b class="xb1"></b></b>
		</div>
   		<div class="xsnazzy">
			<b class="xtop"><b class="xb1"></b><b class="xb2 color_a">
			</b><b class="xb3 color_a"></b><b class="xb4 color_a"></b></b>
			<div class="xboxcontent">
				<h1 class="color_a"><a href="http://202.197.66.228/portal/root/molnet_shbh/index.jsp" target="_blank">分子网络功能注释系统</a></h1>
				<p>1.生物实验数据在线统计学检验分析。<br />2.生物实验数据代谢通路、表达调控等全方位注释信息。<br />3.提供多种信息结果显示，有多种直观图形化的结果显示。</p>
			</div>
			<b class="xbottom"><b class="xb4"></b><b class="xb3"></b>
			<b class="xb2"></b><b class="xb1"></b></b>
		</div>
		<div class="xsnazzy">
			<b class="xtop"><b class="xb1"></b><b class="xb2 color_a">
			</b><b class="xb3 color_a"></b><b class="xb4 color_a"></b></b>
			<div class="xboxcontent">
				<h1 class="color_a"><a href="http://202.197.66.228/portal/root/data_analysis/index.jsp" target="_blank">生物数据分析系统</a></h1>
				<p>1.高通量数据分析、显著性分析、聚类分析、相关分析、主成分分析<br />2.SNP分析、Hardy-Weinberg平衡、LD分析、单体型分析<br />3.医学统计分析、假设检验、方差分析、分参数检验、生存分析、回归分析等。</p>
			</div>
			<b class="xbottom"><b class="xb4"></b><b class="xb3"></b>
			<b class="xb2"></b><b class="xb1"></b></b>
		</div>
      </div>
    </div>
  </div>
    <div id="copyright"> Copyright ©2004-2011&nbsp;<a href="http://www.netgather.com" title="上海网聚化工有限公司">Central South University www.csu.edu.cn</a> All Rights Reserved.
  </div>
</div>


</body>
</html>